java/maven/README.md

```diff
@@ -3,13 +3,13 @@
 ## Building
 
 ```bash
-pack build applications/maven
+pack build applications/maven --builder paketobuildpacks/builder-jammy-base -e BP_JVM_VERSION=17
 ```
 
 Alternatively, if you want to attach a Maven `settings.xml` file to pass additional configuration to Maven.
 
 ```bash
-pack build applications/maven --volume $(pwd)/bindings:/platform/bindings
+pack build applications/maven --builder paketobuildpacks/builder-jammy-base -e BP_JVM_VERSION=17 --volume $(pwd)/bindings:/platform/bindings
```

Comment:
I think setting `BP_JVM_VERSION` is a good change, I have on a number of occasions gotten confused about what version of Java a particular sample is using and had to dig to look it up. Having it explicitly listed in the README is easy.

I would just ask that we consistently include it across all of the Java samples. The JVM version varies across the samples, but we should list whatever it requires. Would you be able to adjust this PR to update all of the Java samples?

I know that's more work, so thanks in advance.